floor_annotation/lib/src/description.dart

```diff
@@ -0,0 +1,7 @@
+/// An annotation used to specify description to show user.
```

Comment:
❓ What's the use-case for this annotation?

Reply:
Sorry for this file, this annotation is from another resource that I am putting to save the enumerates by taking the values of the annotations, without needing a type converter for each enumerate, more details in the other revision